<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFTSignalForm</class>
 <widget class="QWidget" name="FFTSignalForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QwtPlot" name="qwtPlot_peakSignal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>831</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_gidrophone1">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_gidrophone2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>60</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Signal from gidrophone #</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Signal from gidrophone #</string>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot_FFTCycle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>831</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_FFTGidrophone1">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>20</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_FFTGidrophone2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>60</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
